### PR TITLE
Docker_image.rst: remove pip install instruction

### PR DESCRIPTION
**Pull request by raghavgr**
Docker_image.rst: remove pip install instruction

http://docs.coala.io/en/latest/Users/Docker_Image.html#coala-on-gitlab-ci mentions invoking`pip install -r requirements.txt`. It is unnecessary and the documentation has been updated to show the changes.

Closes https://github.com/coala/documentation/issues/486

### CODE REVIEW

**Comment by newbazz** [comment]
LGTM.

**Comment by jayvdb** [comment]
ack https://github.com/coala/documentation/pull/489/commits/f37de4239807b33e734fce3c6a81c008ad77aa76

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
unack

**Comment by jayvdb** [comment]
Please rewrite commit message. "This commit makes sure that the ..." is too wordy - that could be the first words of every commit message, but it is not ... it would be redundant to start every commit message like that.

Describe briefly what is in the commit, and why.

**Comment by jayvdb** [comment]
unack https://github.com/coala/documentation/pull/489/commits/40329da73228464cca86e326f2ff3db1fc2a666f

**Comment by jayvdb** [comment]
(and rebase your commit)

**Comment by raghavgr** [comment]
@jayvdb I have updated my comment. How do I do rebase?

**Comment by jayvdb** [comment]
three commits instead of one.

**Comment by newbazz** [comment]
Why closed this PR? Btw this happened with me also :P

**Comment by vipuldcoder** [comment]
Yes, I will do it
